src/gtsam_tags_node.cpp

```diff
@@ -56,6 +57,7 @@ class LocalizerRunner {
   DataPublisher dataPublisher;
   ConfigListener configListener;
   std::vector<CameraListener> cameraListeners;
+  std::deque<CameraVisionObservation> tooNewCameraObservations;
```

Comment:
I think this queue belongs in the Localizer instead of here in main? How bad would it be to move it?

Reply:
probably not too bad. i'll get to it once i test it all out